styles.js

```diff
@@ -90,3 +92,38 @@ export const NoData = styled.div`
   align-items: center;
   box-shadow: 1px 1px 2px;
 `;
+
+export const SelectionContainer = styled.section`
+  background-color: rgb(255, 255, 255, 0.1);
+  border-radius: 10px;
+  padding: 1rem;
+`;
+
+export const GamesContainer = styled.section`
+  border-radius: 10px;
+  padding: 1.5rem;
+  margin: 0.2rem;
+`;
+
+export const StyledSelect = styled(Select)`
+  width: 100px;
+  height: 40px;
+  font-size: 16px;
```

Comment:
You should not set the font-size in px. [Learn more about the reason here.](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#accessibility-considerations-5)